en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md

```diff
@@ -274,7 +280,6 @@ You can manage OAuth keys either with the Choreo built-in identity provider, Asg
          2. Add the following as authorized redirect URL.
          3. Specify the following as authorized redirect URLs:
          4. Specify the access token type as JWT.
-         5. Set the refresh token expiry time to 1 day. 
 
             !!! tip
```

Comment:
this tip is duplicated isn't it?

Reply:
This is in a tab view. Similar tips with slight variations (Indicating how to perform the role to group mapping for each type of IdP) is displayed in each tab